src/core/agent/agent.ts

```diff
@@ -225,28 +235,34 @@ class Agent {
     }
   }
 
-  async getAgentUrl(): Promise<Partial<AgentUrls>> {
+  async getAgentUrls(): Promise<Partial<AgentUrls>> {
```

Comment:
We just need to get the connect URL here, and if it's not stored. We throw an error.

Reply:
We don't need this function. The UI can call `this.basicStorageService.findById` itself